testing/android_background_image/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,10 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="dev.flutter.android_background_image">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
     <uses-permission android:name="android.permission.INTERNET" />
     <application
         android:name="io.flutter.app.FlutterApplication"
-        android:allowBackup="false"
```

Comment:
Removed because of a new lint that was triggered by the AGP upgrade. There were steps to migrate if the functionality was needed, but I'm not aware of why we would need this so I simply removed it.